src/api/v2/ResourceGroupsExtensionManager.ts

```diff
@@ -42,29 +47,29 @@ export class ResourceGroupsExtensionManager {
     }
 
     async activateApplicationResourceProviders(): Promise<void> {
-        const inActiveResourceContributors =
+        const inactiveResourceContributors =
             vscode.extensions.all
```

Comment:
It might make little difference on performance, but VSCode has a fairly large number of built-in extensions (with publisher `vscode`) that would never be contributing things to this view. Should we add one extra `.filter()` statement to exclude those, to save a little time spent looking into package manifests? Basically equivalent to:
https://github.com/microsoft/vscode-azureresourcegroups/blob/main/src/utils/ExtensionActivationManager.ts#L53

Also, can we flip the `.filter()` for inactive to _before_ the mapping, also for the slight perf gain?

Reply:
Agree it likely won't make a significant performance difference, but makes sense just the same. Added both.